scripts/generate_dump

```diff
@@ -698,16 +1003,25 @@ main() {
     save_cmd "sysctl -a" "sysctl"
 
     save_ip_info
+
+    save_bridge "fdb show" "fdb"
+    save_bridge "vlan show" "vlan"
+
+    save_frr_info
     save_bgp_info
 
     save_cmd "show interface status -d all" "interface.status"
-    save_cmd "show interface counters -d all" "interface.counters"
     save_cmd "show interface transceiver presence" "interface.xcvrs.presence"
     save_cmd "show interface transceiver eeprom --dom" "interface.xcvrs.eeprom"
+    save_cmd "show ip interface" "ip.interface"
 
     save_cmd "lldpctl" "lldpctl"
+    save_cmd "docker exec -it lldp lldpcli show statistics" "lldp.statistics"
+    save_cmd "docker logs bgp" "docker.bgp.log"
+    save_cmd "docker logs swss" "docker.swss.log"
```

Comment:
lldp, bgp, swss services in multi ASIC platform have asic ID appended in the end.
bgp<asicID>, swss<asicID>, lldp<asicID>
eg. bgp0, bgp1...

Reply:
Done